sunpy/image/transform.py

```diff
@@ -116,7 +120,7 @@ def affine_transform(image, rmatrix, order=3, scale=1.0, image_center=None,
 
         if issubclass(image.dtype.type, numbers.Integral):
             warnings.warn("Integer input data has been cast to float64, "
-                          "which is required for the skikit-image transform.",
+                          "which is required for the scikit-image transform.",
```

Comment:
Can you remove the "which is required..." from this warning?  It's not actually a requirement.